docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
<p>It might be obvious to you, but we are skipping steps here. Starting with the upper equation, I would recommend adding the following explicit steps</p><p> </p><ol><li>In the inner parentheses, you factor out <code>1/sigma_j,m+k</code> </li><li>In the inner parantheses, you multiply by a factor of <code>sigma_j,m/sigma_j,m</code> , which is basically multiplying by <code>1</code>   </li><li>Resolve/simplify the denominators inside of the parentheses</li><li>Finally, move the factors from inside the parentheses to outside of the square root</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Yeah... I did that in the previous version as it was not easy for me to understand it before. But then, I skipped that step. I do not know  why? Maybe, because I saw it the second time and I was comfortable with it.  

Your are correct! I should have kept that....I will apply the steps in next set of commits.